apps/mobile/src/features/Transactions/useCases/UtxoList/useUtxoList.ts

```diff
@@ -26,7 +26,6 @@ export const useUtxoList = () => {
   useWalletEvent(wallet, 'utxos', () => queryClient.invalidateQueries(queryKey))
 
   const query = useQuery({
```

Comment:
if before was relying on suspense for the loading state

```suggestion
  const query = useSuspenseQuery({
```

Reply:
It wasn't